developer-workflow/c-api.rst

```diff
@@ -213,7 +213,7 @@ use this API reliably:
   we shouldn't break it *unnecessarily*.
 
 
-Moving an API from the public tier to Unstable
+Moving an API from the public tier to unstable
```

Comment:
I think this should be reverted, since "Unstable" is capitalized in the text below too.
```suggestion
Moving an API from the public tier to Unstable
```